python/cli/prelude_cli/views/iam.py

```diff
@@ -102,117 +116,152 @@ def detach_oidc(controller):
         return controller.detach_oidc()
 
 
-@iam.command("account")
+@iam.command("oidc-name")
+@click.argument("slug")
 @click.pass_obj
 @pretty_print
-def describe_account(controller):
-    """Get account details"""
-    with Spinner(description="Fetching account details"):
-        return controller.get_account()
+def get_oidc_name(controller, slug):
```

Comment:
nit: probably dont need this in the CLI. the lambda only exists so the FE can redirect login to the correct IDP

Reply:
nah, no need